docs/getting-started/release-notes/v2_2_0.md

```diff
@@ -41,9 +45,27 @@ Start here:
 Start here:
 ### Zowe CLI
 
+Zowe CLI and related components contain the following bug fixes.
+
 #### Zowe CLI (Core)
 
-Start here:
+- Updated Imperative to address `ProfileInfo` related issues.
+- Fixed the Zowe Daemon binary exiting with an error if the daemon server does not start within 3 seconds.
+-  Updated Imperative to fix CLI commands failing with error "Cannot find module 'ansi-colors'".
```

Comment:
This line can be removed for changes between Zowe 2.1 and 2.2, since Zowe 2.1 bundled Zowe CLI 7.2.2 and the "Cannot find module 'ansi-colors'" bug only applied to v7.3.0.